REDUNDANCY.md

```diff
@@ -0,0 +1,85 @@
+# Ravenna RTP stream redundancy
+
+## Introduction
+
+Redundancy in a Ravenna RTP stream is a feature that is facilitated by transmitting
+the same audio data over two separate network paths. This is useful in situations
+where users want to ensure that the audio data is transmitted even if one of the
+network paths fails.
+
+As the kernel module and the userspace daemon are currently only capable of handling
+a single network device, redundancy must be handled on a different level. One
+approach to achieve this is to leverage the `nftables` kernel network framework.
+
+**Note** for the below approach to work, make sure that the kernel driver you are
+using has [this commit](https://github.com/bondagit/ravenna-alsa-lkm/commit/e54346da729eeb80a0f88e2539e92ea0657a4bf4).
+
+## Playback (egress)
+
+### Create a dummy network interface
+
+The first step is to create a dummy network interface. This is a virtual network
+interface that discards all packets sent to it. This is sufficient for our purposes,
+as we will capture and treat the packets before they end up at the interface.
+
+```bash
+ip link add name ravenna-dummy type dummy
+ip link set dummy0 up
```

Comment:
You named the device `ravenna-dummy`, so this line should probably read

     ip link set ravenna.-dummy up

Reply:
True, thanks for spotting this. Fixed.